tools/find_optional_imports.py

```diff
@@ -29,7 +29,7 @@ def _main():
         "ipywidgets",
         "scipy.stats",
         "matplotlib",
-        "qiskit.providers.aer",
```

Comment:
We should probably have both TBH as either path will work. While it's probably pretty unlikely that someone will import `qiskit.providers.aer` in a backport PR protecting against it doesn't add any real overhead.

Reply:
Sounds sensible, done in ef656370c3.